core/lib/tasks/exchanges.rake

```diff
@@ -48,7 +48,7 @@ namespace :exchanges do
         # the order builds a shipment on its own on transition to delivery, but we want
         # the original exchange shipment, not the built one
         order.shipments.destroy_all
-        shipments.each { |shipment| shipment.update!(order_id: order.id) }
+        shipments.each { |shipment| shipment.update(order_id: order.id) }
```

Comment:
@damianlegawiec For some reason `shipment.update!` with a bang causes `NoMethodError Exception: undefined method `canceled?' for {:order_id=>4}:Hash` which is situated in state-machines gem. I't won't allow update shipment with bang (!) not only here, but anywhere. I'm not sure is it acceptable here without the exclamation mark.